modules/microshift-oc-mirror-config-cluster.adoc

```diff
@@ -0,0 +1,109 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-oc-mirror-config-cluster_{context}"]
+= Configuring your {microshift-short} cluster to use Operators selected with oc mirror
+
+After you mirror your image set to the mirror registry, you must apply the generated `CatalogSource` custom resource (CR) into the cluster. The `CatalogSource` CR is used by Operator Lifecycle Manager (OLM) to retrieve information about the available Operators in the mirror registry. You can then create a Subscription custom CR and apply that.
+//Q: do we need to do anything else with the cri-o config at this point?
+
+.Prerequisites
+
+* You have mirrored the image set to your registry mirror.
+
+.Procedure
+
+//In our example for deploying operators in the global namespace, we say this value must be set to restricted for microshift; if the new PR merges, will we change `baseline` to `privileged`?
```

Comment:
The default will be `baseline` or it will remain `restricted`? If we change the default to `baseline`, do we still need to change `baseline` to `privileged` in the CatalogSource CR?

Reply:
default will be baseline:
```
➜ oc describe namespaces openshift-marketplace | grep pod-security
              pod-security.kubernetes.io/enforce=baseline
              pod-security.kubernetes.io/enforce-version=v1.24
```
When https://github.com/openshift/microshift/pull/3076 merges, there will be no need to add following to the CatalogSource:
```
  grpcPodConfig:
    securityContextConfig: restricted
```
And following CatalogSource will work out of the box:
```
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: redhat-catalog
  namespace: openshift-marketplace
spec:
  sourceType: grpc
  image: registry.redhat.io/redhat/redhat-operator-index:v4.15
  updateStrategy:
    registryPoll:
      interval: 60m
```